test/TestCase.cpp

```diff
@@ -139,3 +138,30 @@ EVMVersionRestrictedTestCase::EVMVersionRestrictedTestCase(std::string const& _f
 	if (!comparisonResult)
 		m_shouldRun = false;
 }
+
+void EVMVersionRestrictedTestCase::processBytecodeFormatSetting()
+{
+	std::optional<uint8_t> eofVersion = solidity::test::CommonOptions::get().eofVersion();
+	// TODO: Update if EOF moved to Osaka
+	// EOF only available since Prague
+	solAssert(!eofVersion.has_value() ||solidity::test::CommonOptions::get().evmVersion() >= langutil::EVMVersion::prague());
+
+	std::string bytecodeFormatString = m_reader.stringSetting("bytecodeFormat", "legacy");
+	if (bytecodeFormatString == "legacy,>=EOFv1" || bytecodeFormatString == ">=EOFv1,legacy")
+		return;
+
+	// TODO: This is naive implementation because for now we support only one EOF version.
+	if (bytecodeFormatString == "legacy" && eofVersion.has_value())
+		m_shouldRun = false;
+	else if (bytecodeFormatString == ">=EOFv1" && !eofVersion.has_value())
+		m_shouldRun = false;
+	else
+		BOOST_THROW_EXCEPTION(std::runtime_error{"Invalid bytecodeFormat flag: \"" + bytecodeFormatString + "\""});
```

Comment:
CI is failing on this. It's because `legacy` and `>=EOFv1` with the `eofVersion` value that's not skipped are falling into the `else` that throws and they should be handled by skipping it.

Reply:
Fixed